extensions/positron-python/src/test/mocks/pst/index.ts

```diff
@@ -0,0 +1,10 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (C) 2023-2024 Posit Software, PBC. All rights reserved.
```

Comment:
Nope, bad copy paste.
```suggestion
 *  Copyright (C) 2024 Posit Software, PBC. All rights reserved.
```